src/components/reusable/pagetitle/pageTitle.ts

```diff
@@ -0,0 +1,90 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import PageTitleScss from './pageTitle.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+
+/**
+ * Page Title
+ */
+
+@customElement('kyn-page-title')
+export class PageTitle extends LitElement {
+  static override styles = PageTitleScss;
+
+  /** Whether to show headline. */
+  @property({ type: Boolean })
+  showheadLine = false;
+
+  /** Headline text. */
+  @property({ type: String })
+  headLine = '';
+
+  /** Page title text (required). */
+  @property({ type: String })
+  pageTitle = '';
+
+  /** Whether to show subtitle. */
+  @property({ type: Boolean })
+  showSubTitle = false;
+
+  /** Page subtitle text. */
+  @property({ type: String })
+  subTitle = '';
+
+  /** Type of page title `'primary'` & `'secondary'`. */
+  @property({ type: String })
+  type = 'primary';
+
+  /** Whether to show icon. */
+  @property({ type: Boolean })
+  showIcon = false;
+
+  /** The imported carbon icon. Size 56 * 56. Requires `showIcon` true. */
+  @property({ type: Object })
+  icon: any = {};
+
+  override render() {
+    const classes = {
+      'page-title': true,
+      [`page-title-${this.type}`]: true,
+    };
+
+    const subTitleClasses = {
+      'page-subtitle': true,
+      [`page-subtitle-${this.type}`]: true,
+    };
+
+    return html`
+      <div class="page-title-wrapper">
+        ${this.showIcon
+          ? html`<div class="icon-wrapper">
+              <kd-icon .icon=${this.icon} sizeOverride="56"></kd-icon>
```

Comment:
Change the icon to use a `<slot>`. Remember the application may not be using vanilla carbon icons or kd-icon at all.

Reply:
hmm will provide `<slot name="icon"></slot>` instead of `<kd-icon>`

Reply:
one thing is UX wants icon to be 56 * 56 so that's why I put directly <kd-icon> here. If we are providing slots then consumer gonna put any size icons at their end

Reply:
That's ok, we just have to provide the icon usage guidelines.

Reply:
Done